# Binary state control for doors

string group_name
bool state
----